File: src/Domain/Entities/Project.cs
namespace DAOSS.Domain.Entities;

public class Project : BaseEntity
{
	public string Name { get; set; } = string.Empty;
	public string Description { get; set; } = string.Empty;
	public Guid OwnerId { get; set; }
	public Guid DefaultLanguageId { get; set; }
	public string Visibility { get; set; } = "private";
}

File: FlowchartEditorClient/app/src/main/java/com/example/flowcharteditorclient/Project.java
package com.example.flowcharteditorclient;

public class Project {
    public String id; // UUID from server
    public String name;
    public String description;
    public String ownerId;
    public String defaultLanguageId;
    public String visibility;
    public String requiredReviewersRules;
    // role is not part of Project entity, it's in ProjectMember
    // Keeping it for UI compatibility, but it should be fetched separately
    public String role;
}
